### ISSUE TITLE
Built-in NLP Support

### ISSUE THREAD

**Issue by TomVoxsen** [CasperLaiTW/laravel-fb-messenger#89]
Hi Casper,

Could you add get and set functions for handling messages when built-in NLP is enabled? 
https://developers.facebook.com/docs/messenger-platform/built-in-nlp

This did the trick for me
```
/home/tom/projects/laravel-fb-messenger/src/Messages/ReceiveMessage.php:
   63       * @var array
   64       */
   65:     private $nlp = [];
   66  
   67      /**
   ..
  245  
  246      /**
  247:      * Set nlp
  248       *
  249:      * @param $nlp
  250       *
  251       * @return $this
  252       */
  253:     public function setNlp($nlp)
  254      {
  255:         $this->nlp = $nlp;
  256          return $this;
  257      }
  258      /**
  259:      * Get nlp
  260       *
  261       * @return array
  262       */
  263:     public function getNlp()
  264      {
  265:         return $this->nlp;
  266      }
  267  

/home/tom/projects/laravel-fb-messenger/src/Messages/Receiver.php:
   78                      )
   79                      ->setAttachments(Arr::get($message, 'message.attachments', []))
   80:                     ->setNlp(Arr::get($message, 'message.nlp', []));
   81              }
   82 
```
Cheers

**Comment by CasperLaiTW** [comment]
Cool, I will add in the weekend :)

Thanks

**Comment by TomVoxsen** [comment]
Cheers dude!

**Comment by CasperLaiTW** [comment]
I released new version v1.6.0

If any issue, please send to me. 

Thank you
